target/linux/ramips/image/mt7621.mk

```diff
@@ -105,6 +105,11 @@ define Build/ubnt-erx-factory-image
 	fi
 endef
 
+define Build/mitrastarimage
```

Comment:
This should be moved up to maintain alphabetic sorting for Build statements.

Reply:
Done, thanks.